client/components/deposits-overview/next-deposit.tsx

```diff
@@ -140,6 +156,36 @@ const NextDepositDetails: React.FC< NextDepositProps > = ( {
 						</BannerNotice>
 					</div>
 				) }
+
+				{ /* Notice(s) */ }
+				{ ! isLoading && nextDeposit.amount < 0 && (
```

Comment:
@haszari I had a crack at using `interpolateComponents` to include the "Why?" link. I've also added the logic so that this notice is shown when the next deposit is negative, however I'm not 100% convinced if that's correct. It's probably supposed to be check if the available balance is negative, but that will need confirmation.